### ISSUE TITLE
Nature's compass no longer working after buggy world rebuild

### ISSUE THREAD

**Issue by draklorx** [MattCzyr/NaturesCompass#80]
I had an issue on my SMP 1.16.4 forge server where the ender dragon wasn't spawning.

I followed this guide to correct it: https://github.com/AllTheMods/ATM-6/issues/437

In summary the steps I took from that guide are:
1. Backed up my world folder.
2. Regenerated the world with the same seed.
3. Copied level.dat from new world to backed up world and put backed up world back in place.
4. Deleted the DIM1 folder to regenerated The End.

A few weeks later I noticed that the nature's compass was leading to seemingly random places when I'd search for a biome. In some cases it would tell me biomes weren't found when I was standing right next to them. I feel like this is likely related to the rebuild, but I'm not 100% sure. Have you run into anything like this before?

**Comment by draklorx** [comment]
To be clear, I have no record of it working since the server rebuild. It just took a few weeks for me to use the compass again.

And to be more specific I'm using:

forge 1.16.4-35.1.12
NaturesCompass-1.16.4-1.8.6

Here's my full mod list:
 AutoRegLib-1.6-47.jar
 backpacked-1.7.0-1.16.3.jar
 Bookshelf-1.16.4-9.3.18.jar
 buildersaddition-1.16.4-20201203a.jar
 Clumps-6.0.0.12.jar
 cobbler-1.6.1.jar
 comforts-forge-1.16.4-4.0.0.2.jar
 ContentTweaker-1.16-1.0.0_kindlich_manual_build_02.jar
 corpsecomplex-forge-1.16.3-4.0.0.2.jar
 CosmeticArmorReworked-1.16.4-v1.jar
 CraftTweaker-1.16.4-7.1.0_kindlich_manual_build_03.jar
 DungeonCrawl-1.16.3-2.2.1.jar
 dungeons_gear-1.16.4-3.0.4.jar
 EnchantmentDescriptions-1.16.4-6.0.2.jar
 furniture-7.0.0-pre19-1.16.3.jar
 goblintraders-1.2.2-1.16.3.jar
 guardvillagers-1.16.3-1.1.5.jar
 Hwyla-forge-1.10.11-B78_1.16.2.jar
 ilikewood-1.16.3-4.0.2.2.jar
 illagersweararmor-1.0.3.jar
 InitialInventory-5.0.1.11.jar
 jei-1.16.4-7.6.0.62.jar
 JEITweaker-1.16.4-1.0.1.5.jar
 mobcatcher-3.1.jar
 mods_2020_12_12.zip
 NaturesCompass-1.16.4-1.8.6.jar
 OpenLoader-1.16.4-8.0.1.jar
 pamhc2foodcore-1.16.3-1.0.0.jar
 pandoras_creatures-1.16.3-2.0.1.jar
 practicaltools-1.16.3-1.8.jar
 ProgressiveBosses-2.2.0-mc1.16.x.jar
 Quark-r2.4-279.jar
 SoundFilters-0.14_for_1.16.2.jar
 spiders-2.0-1.16.4-1.0.4.jar
 storage_overhaul-1.16.4-1.0.1.jar
 towers_of_the_wild-1.16.4-2.0.1.jar
 valhelsia_structures-16.0.5.jar
 Waystones_1.16.3-7.3.1.jar
 Wyrmroost-1.16.3-1.2.7.jar
 Xaeros_Minimap_20.28.2_Forge_1.16.4.jar

**Comment by draklorx** [comment]
I figured out my issue. When I rebuilt the world I didn't copy over the seed appropriately, so the level.dat file was based on a random seed. This caused all the location data to detect wonky in already generated chunks. To correct this I'm re-rebuilding my world with the correct seed, and then using a world editor to delete most of the banged up overworld chunks.

Nothing to do with this mod. Sorry for the confusion!
